Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "SymEngine"
 uuid = "123dc426-2d89-5057-bbad-38513e3affd8"
-version = "0.8.7"
+version = "0.9"
```

Comment:
Can you keep this at 0.8.7? I'll bump this when we are ready to release.

Reply:
Fixed